src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -0,0 +1,1378 @@
+                case BoundTryStatement boundTryStatement:
+                    return CreateBoundTryStatementOperation(boundTryStatement);
+                case BoundCatchBlock boundCatchBlock:
+                    return CreateBoundCatchBlockOperation(boundCatchBlock);
+                case BoundFixedStatement boundFixedStatement:
+                    return CreateBoundFixedStatementOperation(boundFixedStatement);
+                case BoundUsingStatement boundUsingStatement:
+                    return CreateBoundUsingStatementOperation(boundUsingStatement);
+                case BoundThrowStatement boundThrowStatement:
+                    return CreateBoundThrowStatementOperation(boundThrowStatement);
+                case BoundReturnStatement boundReturnStatement:
+                    return CreateBoundReturnStatementOperation(boundReturnStatement);
+                case BoundYieldReturnStatement boundYieldReturnStatement:
+                    return CreateBoundYieldReturnStatementOperation(boundYieldReturnStatement);
+                case BoundLockStatement boundLockStatement:
+                    return CreateBoundLockStatementOperation(boundLockStatement);
+                case BoundBadStatement boundBadStatement:
+                    return CreateBoundBadStatementOperation(boundBadStatement);
+                case BoundLocalDeclaration boundLocalDeclaration:
+                    return CreateBoundLocalDeclarationOperation(boundLocalDeclaration);
+                case BoundMultipleLocalDeclarations boundMultipleLocalDeclarations:
+                    return CreateBoundMultipleLocalDeclarationsOperation(boundMultipleLocalDeclarations);
+                case BoundLabelStatement boundLabelStatement:
+                    return CreateBoundLabelStatementOperation(boundLabelStatement);
+                case BoundLabeledStatement boundLabeledStatement:
+                    return CreateBoundLabeledStatementOperation(boundLabeledStatement);
+                case BoundExpressionStatement boundExpressionStatement:
+                    return CreateBoundExpressionStatementOperation(boundExpressionStatement);
+                default:
+                    return Operation.CreateOperationNone(boundNode.HasErrors, boundNode.Syntax);
+            }
+        }
+
+        private static IPlaceholderExpression CreateBoundDeconstructValuePlaceholderOperation(BoundDeconstructValuePlaceholder boundDeconstructValuePlaceholder)
+        {
+            bool isInvalid = boundDeconstructValuePlaceholder.HasErrors;
+            SyntaxNode syntax = boundDeconstructValuePlaceholder.Syntax;
+            ITypeSymbol type = boundDeconstructValuePlaceholder.Type;
+            Optional<object> constantValue = ConvertToOptional(boundDeconstructValuePlaceholder.ConstantValue);
+            return new PlaceholderExpression(isInvalid, syntax, type, constantValue);
+        }
+        private static IInvocationExpression CreateBoundCallOperation(BoundCall boundCall)
```

Comment:
generated. styling. let me do once I am done vb side and done with all other code reviews.